docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -206,20 +206,16 @@ predictor_information = predictor.info()
 The `predictor` also remembers what metric predictions should be evaluated with, which can be done with ground truth labels as follows:
 
 ```{.python .input}
-y_pred = predictor.predict(test_data_nolabel)
-perf = predictor.evaluate_predictions(y_true=y_test, y_pred=y_pred, auxiliary_metrics=True)
+y_pred_proba = predictor.predict_proba(test_data_nolabel)
+perf = predictor.evaluate_predictions(y_true=y_test, y_pred=y_pred_proba)
 ```
 
```

Comment:
May be worthwhile to add clarifying statement:

Note that `evaluate_predictions()` requires you to appropriately set `y_pred` as predicted class labels (`predict()` output) or probabilities (`predict_proba()` output) depending on which type of predictions are needed by the `eval_metric`. In contrast, `evaluate()` will automatically handle this for you by internally producing the correct type of predictions for the `eval_metric`.

Reply:
This is no longer necessary because of the refactor. Now there is no downside of always passing prediction probabilities as they will be smartly converted to predictions internally.

Reply:
I meant that they cannot pass predicted classes for metric like AUC, but I guess the error message that pops up should be clear enough.